test/e2e/utils/anpspecbuilder.go

```diff
@@ -28,6 +28,11 @@ type AntreaNetworkPolicySpecBuilder struct {
 	Namespace string
 }
 
+type ANPRuleAppliedToSpec struct {
```

Comment:
there could be some unification between policy AppliedToGroup and rule AppliedToGroup, i.e. it feels like we could use the same type, and that type could be used for `SetAppliedToGroup`, `AddIngress` and `AddEgress` parameters. Same comment for CNP.

Reply:
Will update this in a later commit